is_core/generic_views/table_views.py

```diff
@@ -208,18 +209,30 @@ def _get_list_filter(self):
     def _get_add_url(self):
         return self.core.get_add_url(self.request)
 
+    def is_bulk_change_enabled(self):
+        return hasattr(self.core, 'is_bulk_change_enabled') and self.core.is_bulk_change_enabled()
+
     def get_context_data(self, **kwargs):
         context_data = super(TableView, self).get_context_data(**kwargs)
         context_data.update({
-                                'add_url': self._get_add_url(),
-                                'view_type': self.view_type,
-                                'add_button_value': self.core.model._ui_meta.add_button_verbose_name %
-                                                    {'verbose_name': self.core.model._meta.verbose_name,
-                                                     'verbose_name_plural': self.core.model._meta.verbose_name_plural},
-                                'export_types': self._get_export_types()
-                            })
+            'add_url': self._get_add_url(),
+            'view_type': self.view_type,
+            'add_button_value': self.core.model._ui_meta.add_button_verbose_name % {
+                'verbose_name': self.core.model._meta.verbose_name,
+                'verbose_name_plural': self.core.model._meta.verbose_name_plural},
+            'export_types': self._get_export_types(),
+            'enable_bulk_change': self.is_bulk_change_enabled(),
+            'bulk_change_snippet_name': self.get_bulk_change_snippet_name(),
+            'bulk_change_form_url': self.get_bulk_change_form_url(),
+        })
         return context_data
 
+    def get_bulk_change_snippet_name(self):
+        return '-'.join(('default', self.model._meta.object_name.lower(), 'form'))
+
+    def get_bulk_change_form_url(self):
+        return reverse('IS:bulk-change-' + self.model._meta.object_name.lower())
```

Comment:
If it is manually you shoud define view name inside is core as parameter or method